Add 2024-02-07 EGM results (#5)
redbrick/blog: posts/2024-02-07-egm-results.md

---
title: EGM Results - February 2024
created: 2024-02-07T23:18:39
modified: 2024-02-07T23:27:45
tags:
  - egm
author: distro
---

> Hi all, we'd like to inform you of the outcome of the EGM held today in L125 to amend the constitution and elect and a new Systems Administrator.

Reviewer: ```suggestion
Hi all, we'd like to inform you of the outcome of the EGM held today in L125 to amend the constitution and elect a new Systems Administrator.
```

+ Text in green is any change in proposed new Constitution, for review.
- Text in red is the source article from the current Constitution.
```

### Section 1

```diff
+ The official name of the Society will be Redbrick. This shall appear on correspondance with official bodies
- The official name of the society will be Dublin City University Networking Society. This shall appear on correspondence with official bodies. 
- 1.1 The society operates under the name Redbrick.
```

### Section 2

Section 2.7 on the old constitution was moved to 2.6 on the new one.

```diff
+ 2.3 To create an environment in which its members can develop their skills.
- 2.3 To foster the development of members.

+ 2.4 To provide a forum for members to interact and socialise through activities and discussions which provide experience in computing.
- 2.4 To provide a forum for members to interact and socialise through activities a discuss and gain experience in computer networking.

+ 2.7 To assist other clubs and societies in Dublin City University with computing related questions and tasks.
- 2.7 To increase and promote awareness of electronic communication and related computer systems.
```

### Section 5

> The changes in this section pertain to the election of "technical roles" onto the Redbrick Committee. The three roles have been combined into one as they all fall under the same rules.

Reviewer: ```suggestion
The changes in this section pertain to the election of "technical roles" onto the Redbrick Committee. The election criteria for the three roles have been combined into one, since they all fall under the same rules.
```

+     The outgoing Webmaster, Helpdesk, or Systems Adminstrators OR
+     A third party nominated by the outgoing Webmaster, Helpdesk, or a majority of Systems Administrators OR
+     A committee nominated third party in the case that none of the above criteria can be accommodated.
- 5.3 To be deemed fit for election to the position of Systems Administrator and Webmaster, the candidate must either:
-     Pass an Aptitude test, as devised by the outgoing Webmaster for Webmaster candidates, or system administrators for system administrator candidates.
-     If it is deemed preferable to do so, the test may be set by a third party nominated by the Webmaster (in the case of the Webmaster examination), or by a majority of the system administrators (in the case of the system administrator exams).
-     The current committee will nominate this third party. Be an outgoing Webmaster or Systems Administrator

+ 5.4 The 1st Year Liaison Officer shall be elected at an EGM in semester one, and must be in their first year as an undergraduate student at DCU.
- 5.4 All Helpdesk candidates must sit and pass a test as devised by the outgoing education officer(s)
```

### Section 6

> Ordering of the committee has changed, with "Core" roles being introduced. This means the society is allowed function with just the 4 core members, and is considered functional when it has those 4 members.

Reviewer: ```suggestion
Ordering of the committee has changed, with "Core" roles being introduced. This means the society is allowed to function with just the 4 core members, and is considered functional when it has those 4 members.
```
